### PR TITLE
Feature/figma tokens update picker input for skins

### PR DESCRIPTION
**Pull request by dsv-sergey**

### CODE REVIEW

File: app/src/docs/_examples/pickerInput/PickerWithChangingItemsArray.example.tsx [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
 
 export default function LanguageLevelsArraySinglePicker() {
-    const [pickerValue, setPickerValue] = useState<string>();
-    const [items, setItems] = useState(fullLevelsList);
+    const [pickerValue, setPickerValue] = useState<string>(null);
+    const [items, setItems] = useState([]);
 
     // Memoization, because DataSource should not be recreated on each call.
-    const languageLevelsDataSource = useArrayDataSource({ items }, [items]);
+    const languageLevelsDataSource = useMemo(() => new ArrayDataSource({ items }), [items]);
```
Comment (AlekseyManetov) [inline app/src/docs/_examples/pickerInput/PickerWithChangingItemsArray.example.tsx:24]: why we reworked this to useMemo from useArrayDataSource?

File: app/src/docs/_examples/pickerInput/PickerWithChangingItemsArray.example.tsx [hunk 2]
```diff
@@ -1,17 +1,27 @@
...
+        id: shortLevelsList,
+        caption: 'Short levels list',
+    },
+];
 
 export default function LanguageLevelsArraySinglePicker() {
-    const [pickerValue, setPickerValue] = useState<string>();
-    const [items, setItems] = useState(fullLevelsList);
+    const [pickerValue, setPickerValue] = useState<string>(null);
+    const [items, setItems] = useState([]);
```
Comment (AlekseyManetov) [inline app/src/docs/_examples/pickerInput/PickerWithChangingItemsArray.example.tsx:21]: Why we remove default value? User should work with this example without required selection in multi switch.

File: epam-promo/components/tables/ColumnHeaderDropdown/ColumnHeaderDropdown.tsx [hunk 3]
```diff
@@ -4,7 +4,7 @@ import { Modifier } from "react-popper";
 import { DropdownBodyProps, IDropdownToggler, isMobile, mobilePopperModifier } from '@epam/uui-core';
 import { Dropdown } from '@epam/uui-components';
 import { Panel } from '../../layout';
-import { MobileDropdownWrapper } from "../../pickers";
+import { MobileDropdownWrapper } from "@epam/uui";
 import { SortingPanel, SortingPanelProps } from "./SortingPanel";
 
 type ColumnHeaderDropdownProps = SortingPanelProps & {
```
Comment (AlekseyManetov) [inline epam-promo/components/tables/ColumnHeaderDropdown/ColumnHeaderDropdown.tsx:10]: may be move ColumnHeaderDropdown in UUI also?

File: uui/components/pickers/PickerList.tsx [hunk 4]
```diff
@@ -1,14 +1,22 @@
 import React from 'react';
-import { dataSourceStateToValue, PickerListBase, PickerModalOptions } from '@epam/uui-components';
-import { DataRowProps, IClickable, IHasCaption, IHasPlaceholder, UuiContext, UuiContexts } from '@epam/uui-core';
+import { PickerListBase, PickerModalOptions } from '@epam/uui-components';
+import {
+    DataRowProps,
```
Comment (AlekseyManetov) [inline uui/components/pickers/PickerList.tsx:4]: fix import alignment

**Review by AlekseyManetov — CHANGES_REQUESTED** [review]

File: epam-promo/components/tables/ColumnHeaderDropdown/ColumnHeaderDropdown.tsx (hunk 3 above)
Comment (dsv-sergey) [reply epam-promo/components/tables/ColumnHeaderDropdown/ColumnHeaderDropdown.tsx:10]: It's next iterations. The transfer will be as part of updating the tables

**Pull request merged**